libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -159,8 +160,36 @@ func UseSystemd() bool {
 			}
 		}
 
+		// Assume we have the ability to start a transient unit as a slice
+		// This was broken until systemd v229, but has been back-ported on RHEL environments >= 219
+		// For details, see: https://bugzilla.redhat.com/show_bug.cgi?id=1370299
+		hasStartTransientSliceUnit = true
+
+		// To ensure simple clean-up, we create a slice off the root with no hierarchy
```

Comment:
runC inside a container is a supported config, however in general you aren't running systemd inside that container (so this is a moot point). Though, some people work on running systemd inside containers and might have issues with this. But since it would've been broken before I'm not worried about breaking it twice.